services/centralized-grafana/32.0.2/centralized-grafana.yaml

```diff
@@ -79,4 +79,4 @@ data:
   name: "Grafana"
   dashboardLink: "/dkp/kommander/monitoring/grafana"
   docsLink: "https://grafana.com/docs/"
-  version: "8.2.1"
+  version: "8.3.4"
```

Comment:
for my own edification, where is `8.3.4` coming from?

Reply:
comments added